code/go/0chain.net/conductor/cases/notarising_non_existent_block.go

```diff
@@ -0,0 +1,92 @@
+package cases
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"0chain.net/conductor/conductrpc/stats"
+)
+
+type (
+	// NotarisingNonExistentBlock represents implementation of the TestCase interface.
+	//
+	//	Flow of this test case:
+	//		Notarize not existent block and fetch it
+	//		(T0) Replica_i: send Verification_i_X  0<i<=2f/3 + 1
+	//		(T0 + δ) Replica_1: fetch Block_i
+	//		(T0 + 2*δ) Replica_1: send Notarization_i
+	NotarisingNonExistentBlock struct {
+		serverStats *stats.NodesServerStats
+
+		res *RoundInfo
+
+		wg *sync.WaitGroup
+	}
+)
+
+var (
+	// Ensure NotarisingNonExistentBlock implements TestCase interface.
+	_ TestCase = (*NotarisingNonExistentBlock)(nil)
+)
+
+// NewNotarisingNonExistentBlock creates initialised NotarisingNonExistentBlock.
+func NewNotarisingNonExistentBlock(serverStats *stats.NodesServerStats) *NotarisingNonExistentBlock {
+	wg := new(sync.WaitGroup)
+	wg.Add(1)
+	return &NotarisingNonExistentBlock{
+		serverStats: serverStats,
+		wg:          wg,
+	}
+}
+
+// Check implements TestCase interface.
+func (n *NotarisingNonExistentBlock) Check(ctx context.Context) (success bool, err error) {
+	prepared := make(chan struct{})
+	go func() {
+		n.wg.Wait()
+		prepared <- struct{}{}
+	}()
+
+	select {
+	case <-ctx.Done():
+		return false, ctx.Err()
+
+	case <-prepared:
+		return n.check()
+	}
+}
+
+func (n *NotarisingNonExistentBlock) check() (success bool, err error) {
+	notBlocks := n.res.NotarisedBlocks
+	if len(notBlocks) != 1 || notBlocks[0].Rank != 0 {
```

Comment:
how do we control, that second block from lower ranked miner is not notarized before top ranked?

Reply:
Added mechanism to ignoring blocks generation phase in Generator1 (bbd1159)